src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -63,10 +63,14 @@ public class ExecutionPlanNode
         /// </summary>
         public double RelativeCost { get; set; }
         /// <summary>
-        /// Time take by the node operation in milliseconds
+        /// Time taken by the node operation in milliseconds
         /// </summary>
         public long? ElapsedTimeInMs { get; set; }
         /// <summary>
+        /// CPU Time taken by the node operation in milliseconds
+        /// </summary>
+        public long? ElapsedCpuTimeInMs { get; set; }
```

Comment:
Why do we still need this, if are passing the Rowmetrics array

Reply:
I did it for consistency since elapsed time in milliseconds is being passed at the same level. I can definitely move it into the dictionary, if that's preferable.